refactor-spotify-api.md

```diff
@@ -0,0 +1,24 @@
+
+# Refactor Spofity API Layer + Create better user experience
+
+
+- Today when user enters the app he is prompted to click the "Start Jaming" button twice.
+- This user experience, is a cause of an implementation issue. In other words, not ideal code design.
+- And after the first redirect we are prompted with the same message again, altho we laready dont need to click twice.
+- Technicaly we need 1. redirect the user to spotify url. 2. grab some token from their redirect, for future use.
```

Comment:
After redirecting and grabing I think the access token should be saved in App.state.
Following the storage in App.state I would be able to render the "Start Jammming" button conditionally.